core-parser/src/main/java/uk/ac/ebi/uniprot/parser/tsv/disease/DiseaseEntryMap.java

```diff
@@ -0,0 +1,62 @@
+package uk.ac.ebi.uniprot.parser.tsv.disease;
+
+import uk.ac.ebi.uniprot.common.Utils;
+import uk.ac.ebi.uniprot.cv.disease.CrossReference;
+import uk.ac.ebi.uniprot.cv.disease.Disease;
+import uk.ac.ebi.uniprot.cv.keyword.Keyword;
+import uk.ac.ebi.uniprot.parser.tsv.uniprot.NamedValueMap;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class DiseaseEntryMap implements NamedValueMap {
```

Comment:
Can you please, Create a UnitTests for this class

Reply:
added. thanks